src/components/ReduxAppWrapper/store/reducers/captures.ts

```diff
@@ -3,9 +3,19 @@ import { buildCaptureStateKey } from '~utils/redux'
 
 import type { CaptureActions, CaptureState, CaptureKeys } from '~types/redux'
 
-export const initialState: CaptureState = {}
+export const initialState: CaptureState = {
+  takesHistory: {
+    document_front: [],
+    document_back: [],
```

Comment:
How many takes are we taking? By takes we mean "frames"? For how long do we store them in memory and will we delete them after upload?
I'm wondering if this might be taxing on low end devices memory...

Reply:
Sorry I wasn't making it easy to understand in the first place:

- Firstly, we don't store any "frame" in terms of binary data, but instead, we want to keep track of "how many times the user upload a photo of documents during the current verification session". So there'll barely any memory overhead there, the only data to be store is string timestamps.
- Secondly, (to answer your other question about their names): a "take" means when a document was captured (not necessarily uploaded yet), while a "frame" is more about the data itself. Apparently "takes" are purely for analytics events, something we really want to know about when comparing across user sessions, such as how many times on average a user need to capture & upload a "good quality" document.

Reply:
Thank you!